src/a4/domain/Board.java

```diff
@@ -1,5 +1,71 @@
 package a4.domain;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Board {
+	List<BoardSpace> spaces = new ArrayList<BoardSpace>();
+
```

Comment:
Should board have a size? just a constant variable that says how many spaces there are?